src/app/ubs-admin/components/ubs-admin-employee/employee-form/employee-form.component.html

```diff
@@ -7,18 +7,54 @@ <h6>{{ 'ubs-employee.required-fields' | translate }}</h6>
     <div class="form-group">
       <label class="group-title">{{ 'ubs-employee.name' | translate }}</label>
       <input formControlName="firstName" />
+      <div
+        *ngIf="
+          employeeForm.controls['firstName'].invalid &&
+          (employeeForm.controls['firstName'].dirty || employeeForm.controls['firstName'].touched)
+        "
+        class="text-danger"
+      >
+        <small *ngIf="employeeForm.controls['firstName'].errors.required">{{ 'ubs-employee.required-warning' | translate }}</small>
+        <small *ngIf="employeeForm.controls['firstName'].errors.pattern">{{ 'ubs-employee.name-warning' | translate }}</small>
+      </div>
     </div>
     <div class="form-group">
       <label class="group-title">{{ 'ubs-employee.surname' | translate }}</label>
       <input formControlName="lastName" />
+      <div
+        *ngIf="
+          employeeForm.controls['lastName'].invalid &&
+          (employeeForm.controls['lastName'].dirty || employeeForm.controls['lastName'].touched)
+        "
+        class="text-danger"
+      >
+        <small *ngIf="employeeForm.controls['lastName'].errors.required">{{ 'ubs-employee.required-warning' | translate }}</small>
+        <small *ngIf="employeeForm.controls['lastName'].errors.pattern">{{ 'ubs-employee.name-warning' | translate }}</small>
+      </div>
     </div>
     <div class="form-group">
       <label class="group-title">{{ 'ubs-employee.phone-number' | translate }}</label>
       <input formControlName="phoneNumber" placeholder="+380" [imask]="{ mask: phoneMask }" [unmask]="true" />
+      <div
+        *ngIf="
+          employeeForm.controls['phoneNumber'].invalid &&
+          (employeeForm.controls['phoneNumber'].dirty || employeeForm.controls['phoneNumber'].touched)
+        "
+        class="text-danger"
+      >
+        <small *ngIf="employeeForm.controls['phoneNumber'].errors.required">{{ 'ubs-employee.required-warning' | translate }}</small>
+      </div>
     </div>
     <div class="form-group">
       <label class="group-title">{{ 'ubs-employee.email' | translate }}</label>
       <input formControlName="email" />
+      <div
+        *ngIf="employeeForm.controls['email'].invalid && (employeeForm.controls['email'].dirty || employeeForm.controls['email'].touched)"
+        class="text-danger"
+      >
```

Comment:
Move this > to the previous line

Reply:
I got this issue too.Pritier does it by itself.

Reply:
Prettier move it back